golang.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
language: Go
<!-- lookout footnote separator -->
_If you have feedback about this comment made by the analyzer Dummy, please, tell us: (http://google.es).

-myvalue:NOW YES_